db/seeds.rb

```diff
@@ -10,10 +10,15 @@
 puts '***** Seeding Data Start *****'
 puts '***** Alter Foreign Data Wrapper configurations based on ENV file *****'
 
+test_env = Rails.env == 'test'
+host = test_env ? ENV['FOREIGN_TEST_DB_HOST'] : ENV['FOREIGN_DB_HOST']
+name = test_env ? ENV['FOREIGN_TEST_DB_NAME'] : ENV['FOREIGN_DB_NAME']
+port = test_env ? ENV['FOREIGN_TEST_DB_PORT'] : ENV['FOREIGN_DB_PORT']
+user = test_env ? ENV['FOREIGN_TEST_DB_USERNAME'] : ENV['FOREIGN_DB_USERNAME']
+password = test_env ? ENV['FOREIGN_TEST_DB_PASSWORD'] : ENV['FOREIGN_DB_PASSWORD']
```

Comment:
This could be refactored and collapsed into being more compact, but it would be far less understandable.